src/rpc/wallet_rpc_api.py

```diff
@@ -172,6 +192,7 @@ async def get_next_puzzle_hash(self, request: Dict) -> Dict:
         return response
 
     async def send_transaction(self, request):
+        log.error("rpc_api request: " + str(request))
```

Comment:
This should not be log.error(not an error), change log level locally for debugging